### ISSUE TITLE
Loss function 

### ISSUE THREAD

**Issue by tuxkiller17** [navoshta/behavioral-cloning#4]
Hi,
I want to know your final result value of loss and val loss to compare it with my result
thank s.

**Comment by alexstaravoitau** [comment]
Hey @tuxkiller17 — sorry this has been a while back, and I couldn't find any loss values in my reports. 😔
